<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWindow</class>
 <widget class="QMainWindow" name="ImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="imagesLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="finalImage">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="originalImage">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="stagedImage">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="openOriginalImage">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="openStagedImage">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="openFinalImage">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>168</y>
      <width>190</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="controlLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="sizeSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sizeThrshldLabel">
       <property name="text">
        <string>Size threshold:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="brightnessThrshldLabel">
       <property name="text">
        <string>Brightness threshold:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="brightnessSpin">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="modeBox">
       <item>
        <property name="text">
         <string>Build grid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Restore image</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="modeLabel">
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>191</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="infoLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="compressionCoefficientValue">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="polygonsNumberValue">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="standartDeviationValue">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="compressionCoefficientLabel">
       <property name="text">
        <string>Compression coefficient:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="polygonsNumberLabel">
       <property name="text">
        <string>Polygons number:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="standartDeviationLabel">
       <property name="text">
        <string>Standart deviation:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
